json-spec/examples/landsat8-sample.json

```diff
@@ -35,92 +35,195 @@
  
   "properties": {
     "datetime": "2014-06-02T09:22:02Z",
-    "provider": "LANDSAT_8",
+    "provider": "USGS",
     "license": "PDDL-1.0",
-
-    "eo:off_nadir_angle" : -0.001,
+    "eo:platform": "landsat-8",
```

Comment:
eo:platform should be moved to Collection level, right? Same comment applies to eo:instrument.

Reply:
Yes, you are right, I'll move both of those, thanks